docs/data/base/components/tabs/tabs.md

```diff
@@ -84,18 +84,6 @@ Though not required, you can add the `value` prop to the Tab and Tab Panel to
 </Tabs.Root>
```

Comment:
Not strictly related to this PR, but having a demo on the "Orientation" section instead of just a text would be great

Reply:
We're making an effort to minimise the page length. I think this is less important for an unstyled lib, since there's nothing really to see here except the orientation changing. It only gets interesting when you apply a height restriction, which would only happen in styled land.

So I'll skip this for now but will keep it in mind and maybe revisit once we make other unrelated docs improvements, and get a better sense of how many demos we can add before the pages feel too long.